tailwind.config.cjs

```diff
@@ -1,7 +1,8 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
     content: [
-        "./src/components/TodoList.jsx"
```

Comment:
```content: ['./public/**/*.html', './src/**/*.(jsx|js)']```

Will do the job for all files

Reply:
Changed. Didn't know this trick. Thank you